include/mxnet/c_api.h

```diff
@@ -1399,6 +1399,14 @@ MXNET_DLL int MXNDArrayGetDeferredComputeSymbol(NDArrayHandle *output_handles,
                                                 int num_outputs,
                                                 SymbolHandle *out);
 
+/*!
+ * \brief Clear the info node associated with the arrays.
```

Comment:
The brief is not obvious with the function name.  It is more about how the deferred compute is handled.